priv/repo/migrations/20200213165634_adds_profile_fields_to_users.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Companies.Repo.Migrations.AddsProfileFieldsToUsers do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      add :name, :string
+      add :bio, :text
+      add :interests, :text
+      add :location, :string
```

Comment:
Perhaps we can add a :country field to the table and show a dropdown for countries on the profile form? Can be done in a followup PR though

Reply:
We have the location field from Github, we can probably work with that but we will see how it goes.